src/StudyViewer/StudyDetails.jsx

```diff
@@ -239,7 +273,7 @@ class StudyDetails extends React.Component {
                }}
              />
            </Modal>
-           {(!userHasLoggedIn && !this.props.data.accessRequested) ?
+           {this.requestAccessButtonVisible ?
```

Comment:
i think only check this one is not sufficient, the alert will show after the user has logged in

it should be `(this.requestAccessButtonVisible && !userHasLoggedIn && !this.props.data.accessRequested)`